src/agent/custom_agent.py

```
@@ -22,19 +22,23 @@
-	AgentRunTelemetryEvent,
-	AgentStepTelemetryEvent,
+    AgentEndTelemetryEvent,
+    AgentRunTelemetryEvent,
+    AgentStepTelemetryEvent,
 )
 from browser_use.utils import time_execution_async
 from langchain_core.language_models.chat_models import BaseChatModel
 from langchain_core.messages import (
     BaseMessage,
+    HumanMessage,
+    AIMessage
```

Ruff (0.8.2)

`langchain_core.messages.AIMessage` imported but unused

Remove unused import: `langchain_core.messages.AIMessage`

(F401)